client/app/queue/OrganizationUsers.jsx

```diff
@@ -111,6 +111,7 @@ export default class OrganizationUsers extends React.PureComponent {
       this.setState({
         organizationUsers: [...this.state.organizationUsers, response.body.users.data[0]],
         remainingUsers: this.state.remainingUsers.filter((user) => user.id !== value.id),
+        membershipRequests: this.state.membershipRequests.filter((mr) => parseInt(mr.userId, 10) !== parseInt(value.id, 10)),
```

Comment:
I don't think you need parseInt here. I don't see any reason why this filter function should be any different from the remaningUsers filter right above it.
```suggestion
        membershipRequests: this.state.membershipRequests.filter((request) => request.userId !== value.id),
```

Reply:
we need this. I tested this without parseint, it failed. One is coming in string format, other is coming in integer format. so they are not matching and its not filtering the record from the ui after adding the user.

Reply:
another option is to update the serializer to send the userId in string format. so we can do `.to_s` and remove it here. let me know what you think.

Reply:
Hmm. It does seem to be an integer. We can just leave the parse int in then. The user that is added is serialized with the standard json return vs a fast json serializer like we are using and it seems to send id as a string rather than an integer.